src/main/java/com/impactupgrade/integration/hubspot/builder/ContactBuilder.java

```diff
@@ -30,6 +30,51 @@ public ContactBuilder phone(String phone) {
     return this;
   }
 
+  public ContactBuilder home_phone(String home_phone) {
```

Comment:
camelCase on all method names and arguments (even though HubSpot's API uses snake_case)